Gems/Atom/RHI/Code/Include/Atom/RHI.Reflect/Base.h

```diff
@@ -130,7 +130,23 @@ namespace AZ
             DepthThenKey,
             ReverseDepthThenKey
         };
+        
+        enum class Scaling : uint32_t
+        {
+            None = 0,               // No scaling
+            Stretch,                // Scale the source to fit the target
+            AspectRatioStretch,     // Scale the source to fit the target while preserving the aspect ratio of the source
```

Comment:
does this mode leave grey bars in the areas not drawn, or does it stretch to fill and clip so you don't see the entire rendered content?

Reply:
Ah looks like there are borders containing the window background color https://learn.microsoft.com/en-us/windows/win32/api/dxgi1_2/ne-dxgi1_2-dxgi_scaling